charters/DeiCharter.md

```diff
@@ -0,0 +1,67 @@
+# Diversity Equity and Inclusion Committee Charter
+
+Project Jupyter welcomes and encourages participation in our community by people of all backgrounds and identities. We are committed to promoting and sustaining a culture that values mutual respect, tolerance, and learning. We work together as a community to help each other live out these values.
+
+The Diversity Equity & Inclusion (DEI) Committee is a standing committee established under Jupyter’s governance model. The committee reviews its charter annually and may make changes at any time during the year. Substantive changes to the purpose or mission must be submitted to the Executive Council for approval.
+
+## WHY: Purpose 
+
+Why is the committee being created? How does this group contribute to Jupyter? 
+
+The purpose of this committee is to lead strategic initiatives designed to improve the diversity, equity, and inclusion of Project Jupyter community leaders, core developers, and end users across the globe. For the long-term sustainability of our community, we champion efforts to build an open source environment where people feel respected, valued, and encouraged to continue participating. 
+
+## WHAT: Mission/Areas of Responsibility
+
+What will this committee work on? How will the purpose described above be accomplished?
+
+The DEI standing committee is responsible to: 
+* Propose, execute, and evaluate the impact of DEI initiatives
+* Serve as a point of contact for DEI topics for the global Jupyter community
+* Serve as DEI ambassadors by including this topic in the discussions and spaces within Project Jupyter
+* Recommend and promote training, educational, and motivational materials on DEI 
+* Flag DEI issues or potential issues to the Executive Council (EC) or the Software Steering Council (SSC)
+
+## WHO: Committee Composition
+### Members
+Members should affirm the opening paragraph of this charter and agree to the purpose of this committee
+Committees should consist of at least 2 members serving a 1 year term with an option for unlimited renewal 
```

Comment:
`Committees` (plural) is used in a couple of places in this doc- are there multiple (sub?)committees?